smart-contracts/contracts/DigitalaxAuction.sol

```diff
@@ -276,9 +335,6 @@ contract DigitalaxAuction is Context, ReentrancyGuard {
             delete highestBids[_garmentTokenId];
         }
 
-        // Transfer the token back to the lister
```

Comment:
`require(auction.lister != address(0), "DigitalaxAuction.cancelAuction: Auction does not exist");` This should use the same check throughout - uses dates higher up

Reply:
What does this mean?